README.md

```diff
@@ -83,3 +83,15 @@ Code is auto-formatted with clang-format 10 whenever a git commit is made. Pleas
 Prepare the pre-commit formatting to run like this:
 
 `pre-commit install`
+
+
+## Stuff to add
+
+1. Explation of packages
+2. How to start:
+   - Simple simulation with FakeSystem
+   - Hardware
+
+3. How to test:
+   - Using forward_command_controller
+   - Using joint_trajectory_controller
```

Comment:
When we make a final README PR, let's emphasize `joint_trajectory_controller` and trajectory execution first. I get the impression that's still what 90% of users need.